cmd2/cmd2.py

```diff
@@ -280,6 +281,24 @@ def cmd_wrapper(cmd2_instance, statement: Union[Statement, str]):
     return arg_decorator
 
 
+class _SavedReadlineSettings:
```

Comment:
Does this code belong in rl_utils.py?

Reply:
Since this class is only used in cmd2.py, I'm OK leaving it.